.travis.yml

```diff
@@ -44,6 +46,13 @@ jobs:
           on:
             repo: lbryio/lbrycrd
             all_branches: true
+        - provider: script
+          script: bash packaging/docker-for-binary/auto_deploy.sh ${TRAVIS_BRANCH}
+          skip_cleanup: true
+          on:
+            repo: lbryio/lbrycrd
+            all_branches: true
```

Comment:
This has to be tags only. (You're welcome to tag things in the lbrycrd repo as you see fit.)